### ISSUE TITLE
【EESS】New submissions for Thu, 31 Aug 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#583]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### Modality Cycles with Masked Conditional Diffusion for Unsupervised  Anomaly Segmentation in MRI
 - **Authors:** Ziyun Liang, Harry Anthony, Felix Wagner, Konstantinos Kamnitsas
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2308.16150
 - **Pdf link:** https://arxiv.org/pdf/2308.16150
 - **Abstract**
 Unsupervised anomaly segmentation aims to detect patterns that are distinct from any patterns processed during training, commonly called abnormal or out-of-distribution patterns, without providing any associated manual segmentations. Since anomalies during deployment can lead to model failure, detecting the anomaly can enhance the reliability of models, which is valuable in high-risk domains like medical imaging. This paper introduces Masked Modality Cycles with Conditional Diffusion (MMCCD), a method that enables segmentation of anomalies across diverse patterns in multimodal MRI. The method is based on two fundamental ideas. First, we propose the use of cyclic modality translation as a mechanism for enabling abnormality detection. Image-translation models learn tissue-specific modality mappings, which are characteristic of tissue physiology. Thus, these learned mappings fail to translate tissues or image patterns that have never been encountered during training, and the error enables their segmentation. Furthermore, we combine image translation with a masked conditional diffusion model, which attempts to `imagine' what tissue exists under a masked area, further exposing unknown patterns as the generative model fails to recreate them. We evaluate our method on a proxy task by training on healthy-looking slices of BraTS2021 multi-modality MRIs and testing on slices with tumors. We show that our method compares favorably to previous unsupervised approaches based on image reconstruction and denoising with autoencoders and diffusion models.
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
### Volt/VAR Optimization in the Presence of Attacks: A Real-Time  Co-Simulation Study
 - **Authors:** Mohd Asim Aftab, Astha Chawla, Pedro P. Vergara, Shehab Ahmed, Charalambos Konstantinou
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2308.15797
 - **Pdf link:** https://arxiv.org/pdf/2308.15797
 - **Abstract**
 Traditionally, Volt/VAR optimization (VVO) is performed in distribution networks through legacy devices such as on-load tap changers (OLTCs), voltage regulators (VRs), and capacitor banks. With the amendment in IEEE 1547 standard, distributed energy resources (DERs) can now provide reactive power support to the grid. For this, renewable energy-based DERs, such as PV, are interfaced with the distribution networks through smart inverters (SIs). Due to the intermittent nature of such resources, VVO transforms into a dynamic problem that requires extensive communication between the VVO controller and devices performing the VVO scheme. This communication, however, can be potentially tampered with by an adversary rendering the VVO ineffective. In this regard, it is important to assess the impact of cyberattacks on the VVO scheme. This paper develops a real-time co-simulation setup to assess the effect of cyberattacks on VVO. The setup consists of a real-time power system simulator, a communication network emulator, and a master controller in a system-in-the-loop (SITL) setup. The DNP3 communication protocol is adopted for the underlying communication infrastructure. The results show that corrupted communication messages can lead to violation of voltage limits, increased number of setpoint updates of VRs, and economic loss.
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
### A Neural-enhanced Factor Graph-based Algorithm for Robust Positioning in  Obstructed LOS Situations
 - **Authors:** Alexander Venus, Erik Leitinger, Stefan Tertinek, Klaus Witrisal
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2308.15972
 - **Pdf link:** https://arxiv.org/pdf/2308.15972
 - **Abstract**
 This paper presents a neural-enhanced probabilistic model and corresponding factor graph-based sum-product algorithm for robust localization and tracking in multipath-prone environments. The introduced hybrid probabilistic model consists of physics-based and data-driven measurement models capturing the information contained in both, the line-of-sight (LOS) component as well as in multipath components (NLOS components). The physics-based and data-driven models are embedded in a joint Bayesian framework allowing to derive from first principles a factor graph-based algorithm that fuses the information of these models. The proposed algorithm uses radio signal measurements from multiple base stations to robustly estimate the mobile agent's position together with all model parameters. It provides high localization accuracy by exploiting the position-related information of the LOS component via the physics-based model and robustness by exploiting the geometric imprint of multipath components independent of the propagation channel via the data-driven model. In a challenging numerical experiment involving obstructed LOS situations to all anchors, we show that the proposed sequential algorithm significantly outperforms state-of-the-art methods and attains the posterior Cramer-Rao lower bound even with training data limited to local regions.
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### unORANIC: Unsupervised Orthogonalization of Anatomy and  Image-Characteristic Features
 - **Authors:** Sebastian Doerrich, Francesco Di Salvo, Christian Ledig
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2308.15507
 - **Pdf link:** https://arxiv.org/pdf/2308.15507
 - **Abstract**
 We introduce unORANIC, an unsupervised approach that uses an adapted loss function to drive the orthogonalization of anatomy and image-characteristic features. The method is versatile for diverse modalities and tasks, as it does not require domain knowledge, paired data samples, or labels. During test time unORANIC is applied to potentially corrupted images, orthogonalizing their anatomy and characteristic components, to subsequently reconstruct corruption-free images, showing their domain-invariant anatomy only. This feature orthogonalization further improves generalization and robustness against corruptions. We confirm this qualitatively and quantitatively on 5 distinct datasets by assessing unORANIC's classification accuracy, corruption detection and revision capabilities. Our approach shows promise for enhancing the generalizability and robustness of practical applications in medical image analysis. The source code is available at https://github.com/sdoerrich97/unORANIC.
### The Impact of Reference-Command Preview on Human-in-the-Loop Control  Behavior
 - **Authors:** Pedram Rabiee, S. Alireza Seyyed Mousavi, Amelia J. S. Sheffler, Erik Hellström, Mrdjan Jankovic, Mario A. Santillo, T. M. Seigler, Jesse B. Hoagg
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2308.15633
 - **Pdf link:** https://arxiv.org/pdf/2308.15633
 - **Abstract**
 This article presents results from an experiment in which 44 human subjects interact with a dynamic system to perform 40 trials of a command-following task. The reference command is unpredictable and different on each trial, but all subjects have the same sequence of reference commands for the 40 trials. The subjects are divided into 4 groups of 11 subjects. One group performs the command-following task without preview of the reference command, and the other 3 groups are given preview of the reference command for different time lengths into the future (0.5 s, 1 s, 1.5 s). A subsystem identification algorithm is used to obtain best-fit models of each subject's control behavior on each trial. The time- and frequency-domain performance, as well as the identified models of the control behavior for the 4 groups are examined to investigate the effects of reference-command preview. The results suggest that preview tends to improve performance by allowing the subjects to compensate for sensory time delay and approximate the inverse dynamics in feedforward. However, too much preview may decrease performance by degrading the ability to use the correct phase lead in feedforward.
### Widely Separated MIMO Radar Using Matrix Completion
 - **Authors:** Shunqiao Sun, Yunqiao Hu, Kumar Vijay Mishra, Athina P. Petropulu
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2308.15636
 - **Pdf link:** https://arxiv.org/pdf/2308.15636
 - **Abstract**
 We present a low-complexity widely separated multiple-input-multiple-output (WS-MIMO) radar that samples the signals at each of its multiple receivers at reduced rates. We process the low-rate samples of all transmit-receive chains at each receiver as data matrices. We demonstrate that each of these matrices is low rank as long as the target moves slowly within a coherent processing interval. We leverage matrix completion (MC) to recover the missing samples of each receiver signal matrix at the common fusion center. Subsequently, we estimate the targets' positions and Doppler velocities via the maximum likelihood method. Our MC-WS-MIMO approach recovers missing samples and thereafter target parameters at reduced rates without discretization. Our analysis using ambiguity functions shows that antenna geometry affects the performance of MC-WS-MIMO. Numerical experiments demonstrate reasonably accurate target localization at SNR of 20 dB and sampling rate reduction to 20%.
### Implementation and Evaluation of Physical Layer Key Generation on SDR  based LoRa Platform
 - **Authors:** Yingying Hu, Dongyang Xu, Tiantian Zhang
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2308.15696
 - **Pdf link:** https://arxiv.org/pdf/2308.15696
 - **Abstract**
 Physical layer key generation technology which leverages channel randomness to generate secret keys has attracted extensive attentions in long range (LoRa)-based networks recently. We in this paper develop a software-defined radio (SDR) based LoRa communications platform using GNU Radio on universal software radio peripheral (USRP) to implement and evaluate typical physical layer key generation schemes. Thanks to the flexibility and configurability of GNU Radio to extract LoRa packets, we are able to obtain the fine-grained channel frequency response (CFR) through LoRa preamble based channel estimation for key generation. Besides, we propose a lowcomplexity preprocessing method to enhance the randomness of quantization while reducing the secret key disagreement ratio. The results indicate that we can achieve 367 key bits with a high level of randomness through just a single effective channel probing in an indoor environment at a distance of 2 meters under the circumstance of a spreading factor (SF) of 7, a preamble length of 8, a signal bandwidth of 250 kHz, and a sampling rate of 1 MHz.
### Round-Trip Energy Efficiency and Energy-Efficiency Fade Estimation for  Battery Passport
 - **Authors:** Camiel Beckers (1), Erik Hoedemaekers (1), Arda Dagkilic (2), Henk Jan Bergveld (3 and 4) ((1) TNO - Powertrains Dept., (2) VDL Enabling Transport Solutions, (3) Eindhoven University of Technology - Dept. of Electrical Engineering, (4) NXP Semiconductors)
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2308.15828
 - **Pdf link:** https://arxiv.org/pdf/2308.15828
 - **Abstract**
 The battery passport is proposed as a method to make the use and remaining value of batteries more transparent. The future EU Battery Directive requests this passport to contain the round-trip energy efficiency and its fade. In this paper, an algorithm is presented and demonstrated that estimates the round-trip energy efficiency of a battery pack. The algorithm identifies round trips based on battery current and SoC and characterizes these round trips based on certain conditions. 2D efficiency maps are created as a function of the conditions `temperature' and `RMS C-rate'. The maps are parameterized using multiple linear regression, which allows comparison of the efficiency under the same conditions. Analyzing data from three battery-electric buses over a period of 3.5 years reveals an efficiency fade of up to 0.86 percent point.
### Attention-based CT Scan Interpolation for Lesion Segmentation of  Colorectal Liver Metastases
 - **Authors:** Mohammad Hamghalam, Richard K. G. Do, Amber L. Simpson
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2308.15932
 - **Pdf link:** https://arxiv.org/pdf/2308.15932
 - **Abstract**
 Small liver lesions common to colorectal liver metastases (CRLMs) are challenging for convolutional neural network (CNN) segmentation models, especially when we have a wide range of slice thicknesses in the computed tomography (CT) scans. Slice thickness of CT images may vary by clinical indication. For example, thinner slices are used for presurgical planning when fine anatomic details of small vessels are required. While keeping the effective radiation dose in patients as low as possible, various slice thicknesses are employed in CRLMs due to their limitations. However, differences in slice thickness across CTs lead to significant performance degradation in CT segmentation models based on CNNs. This paper proposes a novel unsupervised attention-based interpolation model to generate intermediate slices from consecutive triplet slices in CT scans. We integrate segmentation loss during the interpolation model's training to leverage segmentation labels in existing slices to generate middle ones. Unlike common interpolation techniques in CT volumes, our model highlights the regions of interest (liver and lesions) inside the abdominal CT scans in the interpolated slice. Moreover, our model's outputs are consistent with the original input slices while increasing the segmentation performance in two cutting-edge 3D segmentation pipelines. We tested the proposed model on the CRLM dataset to upsample subjects with thick slices and create isotropic volume for our segmentation model. The produced isotropic dataset increases the Dice score in the segmentation of lesions and outperforms other interpolation approaches in terms of interpolation metrics.
### The DeepZen Speech Synthesis System for Blizzard Challenge 2023
 - **Authors:** Christophe Veaux, Ranniery Maia, Spyridoula Papendreou
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2308.15945
 - **Pdf link:** https://arxiv.org/pdf/2308.15945
 - **Abstract**
 This paper describes the DeepZen text to speech (TTS) system for Blizzard Challenge 2023. The goal of this challenge is to synthesise natural and high-quality speech in French, from a large monospeaker dataset (hub task) and from a smaller dataset by speaker adaptation (spoke task). We participated to both tasks with the same model architecture. Our approach has been to use an auto-regressive model, which retains an advantage for generating natural sounding speech but to improve prosodic control in several ways. Similarly to non-attentive Tacotron, the model uses a duration predictor and gaussian upsampling at inference, but with a simpler unsupervised training. We also model the speaking style at both sentence and word levels by extracting global and local style tokens from the reference speech. At inference, the global and local style tokens are predicted from a BERT model run on text. This BERT model is also used to predict specific pronunciation features like schwa elision and optional liaisons. Finally, a modified version of HifiGAN trained on a large public dataset and fine-tuned on the target voices is used to generate speech waveform. Our team is identified as O in the the Blizzard evaluation and MUSHRA test results show that our system performs second ex aequo in both hub task (median score of 0.75) and spoke task (median score of 0.68), over 18 and 14 participants, respectively.
